### PR TITLE
[RTM] Fix missing table in install tool

### LINKED ISSUE #219: ContaoDataCollector: don't collect layout data, while table not exists (e.g. during install)
Running the installer at `contao/install` with an initial update database (empty database) ends up in an sql error: 

```
TableNotFoundException in AbstractMySQLDriver.php line 53:
An exception occurred while executing 'SELECT tl_layout.*, j1.id AS pid__id, j1.tstamp AS pid__tstamp, j1.name AS pid__name, j1.author AS pid__author, j1.folders AS pid__folders, j1.screenshot AS pid__screenshot, j1.templates AS pid__templates, j1.vars AS pid__vars FROM tl_layout LEFT JOIN tl_theme j1 ON tl_layout.pid=j1.id WHERE tl_layout.id=NULL LIMIT 0,1':

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'contaosf.tl_layout' doesn't exist
```

Partial (important) call stack part:
```
1. in AbstractMySQLDriver.php line 53
2. at AbstractMySQLDriver->convertException('An exception occurred while executing 'SELECT tl_layout.*, j1.id AS pid__id, j1.tstamp AS pid__tstamp, j1.name AS pid__name, j1.author AS pid__author, j1.folders AS pid__folders, j1.screenshot AS pid__screenshot, j1.templates AS pid__templates, j1.vars AS pid__vars FROM tl_layout LEFT JOIN tl_theme j1 ON tl_layout.pid=j1.id WHERE tl_layout.id=NULL LIMIT 0,1': SQLSTATE[42S02]: Base table or view not found: 1146 Table 'contaosf.tl_layout' doesn't exist', object(PDOException)) in DBALException.php line 116
3. at DBALException::driverExceptionDuringQuery(object(Driver), object(PDOException), 'SELECT tl_layout.*, j1.id AS pid__id, j1.tstamp AS pid__tstamp, j1.name AS pid__name, j1.author AS pid__author, j1.folders AS pid__folders, j1.screenshot AS pid__screenshot, j1.templates AS pid__templates, j1.vars AS pid__vars FROM tl_layout LEFT JOIN tl_theme j1 ON tl_layout.pid=j1.id WHERE tl_layout.id=NULL LIMIT 0,1', array()) in Connection.php line 836
4. at Connection->executeQuery('SELECT tl_layout.*, j1.id AS pid__id, j1.tstamp AS pid__tstamp, j1.name AS pid__name, j1.author AS pid__author, j1.folders AS pid__folders, j1.screenshot AS pid__screenshot, j1.templates AS pid__templates, j1.vars AS pid__vars FROM tl_layout LEFT JOIN tl_theme j1 ON tl_layout.pid=j1.id WHERE tl_layout.id=NULL LIMIT 0,1') in Statement.php line 284
5. at Statement->query() in Statement.php line 257
6. at Statement->execute(null) in Model.php line 1080
7. at Model::find(array('limit' => '1', 'column' => 'id', 'value' => null, 'return' => 'Model')) in Model.php line 785
8. at Model::findByPk(null) in ContaoDataCollector.php line 229
9. at ContaoDataCollector->addSummaryData() in ContaoDataCollector.php line 69
10. at ContaoDataCollector->collect(object(Request), object(Response), object(DriverException)) in Profiler.php line 209
```

### PR DESCRIPTION
**Pull request by aschempp**
fixes #219

### CODE REVIEW

File: src/DataCollector/ContaoDataCollector.php [hunk 1]
```diff
@@ -217,24 +217,16 @@ private function addSummaryData()
...
-            if (null !== ($layoutModel = LayoutModel::findByPk($objPage->layout))) {
-                $layout = sprintf('%s (ID %s)', $layoutModel->name, $layoutModel->id);
-            }
         }
 
         $this->data['summary'] = [
             'version'        => $this->getContaoVersion(),
             'scope'          => $this->getContainerScope(),
-            'layout'         => $layout,
+            'layout'         => $this->getLayoutName() ?: 'N/A',
```
Comment (leofeyer) [inline src/DataCollector/ContaoDataCollector.php]: Why don't you simply return `'N/A'` in the `getLayoutName()` method?

Comment (aschempp) [inline src/DataCollector/ContaoDataCollector.php]: Thought about that too, but `N/A` is not a layout name…

Comment (leofeyer) [inline src/DataCollector/ContaoDataCollector.php]: You are being picky :smile:

**Comment by aschempp** [comment]
I have added a little improvement related to the DataCollector.

**Pull request merged**